package.json

```diff
@@ -119,7 +119,7 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "atom-babel6-transpiler": "^1.1.1",
+    "atom-babel6-transpiler": "^1.1.3",
     "atom-linter": "^10.0.0",
     "atom-package-deps": "^4.6.0",
     "babel-preset-node5": "^12.0.0",
```

Comment:
Can we update this one too? We should be transpiling to 7.4.

Reply:
Moving to `babel-preset-env` is something I want to do in a separate PR.